.github/workflows/autoupdate.yml

```diff
@@ -84,13 +84,15 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.PAT }}
         run: |
+          errors=""
           REPOS=$(planemo ci_find_repos "${{ matrix.upstream_repo_dir }}")
           for REPO in $REPOS; do
             echo $REPO
             # First try to update
             echo "Running autoupdate command..."
             cd "$REPO"
-            planemo autoupdate . --skiplist "${{ github.workspace }}/autoupdate/${{ matrix.upstream_repo_owner }}_${{ matrix.upstream_repo_name }}_skip_list" > "${{ github.workspace }}/autoupdate.log"
+            # This may fail
+            planemo autoupdate . --skiplist "${{ github.workspace }}/autoupdate/${{ matrix.upstream_repo_owner }}_${{ matrix.upstream_repo_name }}_skip_list" > "${{ github.workspace }}/autoupdate.log" || errors="$error\nCannot autoupdate $REPO"
```

Comment:
You are using sometimes `errors` and sometimes `error` as a variable name, I don't think that's on purpose?

Reply:
Oops good catch.